### DISCUSSION TITLE
How can i send a payment with an external button?

### DISCUSSION THREAD

**Discussion by joxpulp** [mercadopago/sdk-react#28] [Q&A] [answered]
Using cardPayment Brick, there is an option to hide the default paymentButton provided by the package, but my concern is how can I recover the data to send my payment with my custom button?

Also I think that there is a bug with cardPayment brick, when i refresh the page I have two instance of the cardPayment brick rendered:

https://user-images.githubusercontent.com/73070103/231498501-f98ad79e-d478-4022-bd75-95bd813926c7.mp4

**Comment by AdaltonLeite** [comment, marked as answer]
Hi @joxpulp, sorry for the delay in replying.

About your questions, yes it is possible to recover the data using the `cardFormBrick` instance that is allocated in the `window`. You can see an example of how to do this [here](https://github.com/mercadopago/sdk-react/blob/main/examples/bricks/CardPayment/1-Customization.tsx#L22). 

About the bug, we fixed it in the new version latest [0.0.8](https://www.npmjs.com/package/@mercadopago/sdk-react/v/0.0.8), you can update your lib and you will see that the error doesn't happen anymore. 

Thanks for the report and I hope that helps, thank you!

**Reply by joxpulp** [reply]
Thanks 🙌🏽
